bin/compile

```diff
@@ -189,8 +191,6 @@ cp -R "$CACHE_DIR/.scalingo/python-stack" .scalingo/ &> /dev/null || true
 cp -R "$CACHE_DIR/.scalingo/python-version" .scalingo/ &> /dev/null || true
 # A plain text file which contains the current sqlite3 version being used (used for cache busting).
 cp -R "$CACHE_DIR/.scalingo/python-sqlite3-version" .scalingo/ &> /dev/null || true
-# Any pre-compiled binaries, provided by the buildpack.
-cp -R "$CACHE_DIR/.scalingo/vendor" .scalingo/ &> /dev/null || true
```

Comment:
By removing this we must be sure the new stack is deployed with `jq` in it.

Reply:
Nice catch 👍🏾